src/cli.js

```diff
@@ -36,6 +47,22 @@ function checkInternet() {
 }
 
 if (require.main === module) {
+  const sanitizedArgs = [];
```

Comment:
@tribut but you need to use the value at line 237: `.parse(process.argv);` should become `.parse(sanitizedArgs);`

That suggests you didn't test these last changes. Can you do a quick check and validate that all three options work as expected? Thanks 🙂.